src/components/bildval/guessOption.tsx

```diff
@@ -39,9 +39,11 @@ export const BildvalGuessOption = ({
       // Check if correct
       if (solution?.id === guess.id) {
         // Show confetti
-        setSuccess(true);
+        setShowConfetti(true);
         // Play success audio
         playSuccessAudio?.();
+        // Hide Confetti with a delay of 500 ms
+        setTimeout(() => setShowConfetti(false), 500);
```

Comment:
The delay timing is a bit short, the confetti ends abruptly.

Suggest increasing to 1.5-2s till the confetti has subsided, thanks

Reply:
Sure, I have increased the delay to 1.5s.